src/cljdoc/render/rich_text.clj

```diff
@@ -37,7 +37,8 @@
     :as _opts}]
   (.. (HtmlRenderer/builder
        (doto (MutableDataSet.)
-         (.set AnchorLinkExtension/ANCHORLINKS_ANCHOR_CLASS "md-anchor")))
+         (.set AnchorLinkExtension/ANCHORLINKS_ANCHOR_CLASS "md-anchor")
+         (.set HtmlRenderer/FENCED_CODE_NO_LANGUAGE_CLASS "language-clojure")))
```

Comment:
Nice!

Reply:
Yeah, we are in luck such option exists :)